projects/aca-content/src/lib/components/search/search-results-row/search-results-row.component.ts

```diff
@@ -46,6 +46,8 @@ import { MatDialogModule } from '@angular/material/dialog';
   host: { class: 'aca-search-results-row' }
 })
 export class SearchResultsRowComponent implements OnInit, OnDestroy {
+  private readonly highlightPrefix = "<span class='aca-highlight'>";
```

Comment:
maybe we should keep private readonly separately than private (so just have empty line between them)

Reply:
Done